layrry-launcher/src/main/java/org/moditect/layrry/launcher/internal/Args.java

```diff
@@ -26,7 +26,7 @@
     @Parameter(description = "[arguments]")
     private List<String> mainArgs = new ArrayList<>();
 
-    @Parameter(names = "--layers-config", required = true, description = "Layers configuration file. May be a local file or remote URL.")
+    @Parameter(names = "--layers-config", description = "Layers configuration file. May be a local file or remote URL.")
```

Comment:
Can we add some examples perhaps? Not sure what's the best place, the option description itself, or maybe the overall tool description (assuming there is such thing)?

Reply:
Done. Usage now looks like

```sh
Missing --layers-config parameter or local file named layers[.yml|.toml]

Usage: org.moditect.layrry.launcher.LayrryLauncher [options] [arguments]
  Options:
    --basedir
      Base directory for resolving relative paths
    --help
      Show usage and quit
    --layers-config
      Layers configuration file. May be a local file or remote URL. Examples: 
      path/to/layers.yml, https://server/path/to/layers.toml
    --properties
      Additional configuration properties. May be a local file or remote URL. 
      Must use the Java properties format. Examples: 
      path/to/versions.properties, https://server/path/to/versions.properties
    --version
      Show version and quit

Supported config formats are [.yml, .toml]
```

Reply:
A, that's perfect!

Reply:
The one thing we might want to revise later on is the FQN right at the top, as that'd be confusing when e.g. used via sdkman or jbang. But we can leave that for another time.

Reply:
The FQN was added recently with https://github.com/moditect/layrry/commit/47a4d851dd5cbd0fb0b9117a51002bdeca4034fd as it displayed `<main-class>` before.